backend/src/main/java/com/htecgroup/skynest/filter/CustomAuthenticationFilter.java

```diff
@@ -89,8 +89,13 @@ protected void successfulAuthentication(
     String token =
         JwtUtils.generate(
             user.getUsername(), JwtUtils.ACCESS_TOKEN_EXPIRATION_MS, "roles", authorities);
+    String refresh_token =
+        JwtUtils.generate(
```

Comment:
There should be new method for this when you take the latest development version